src/ui/public/vis/vis_types/react_vis_type.js

```diff
@@ -19,49 +19,44 @@
 
 import React from 'react';
 import { render, unmountComponentAtNode } from 'react-dom';
-import { VisTypeProvider } from './';
+import chrome from '../../chrome';
+import { VisType } from './base_vis_type';
 
-export function ReactVisTypeProvider(Private, getAppState, config) {
-  const VisType = Private(VisTypeProvider);
-
-  class ReactVisController {
-    constructor(el, vis) {
-      this.el = el;
-      this.vis = vis;
-    }
-
-    render(visData, updateStatus) {
-      this.visData = visData;
+class ReactVisController {
+  constructor(el, vis) {
+    this.el = el;
+    this.vis = vis;
+  }
```

Comment:
I like the el-vis 🕺🏻naming, but maybe we can say use  (element, vis) for a bit more clear naming